website/docs/reference/warehouse-profiles/materialize-profile.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "ClickHouse Profile"
+---
+
+:::info Vendor-supported plugin
+
+Certain core functionality may vary. If would like to report a bug, request a feature, or contribute, you can see the source code and open an issue in the repository below.
```

Comment:
Could we say something more along the lines of go to the linked repository + then open an issue for the maintainers? See the source code sounds off to me.